src/Test/contlinear.jl

```diff
@@ -1540,6 +1540,52 @@ function linear14test(model::MOI.ModelLike, config::TestConfig)
     end
 end
 
+# Empty vector affine function rows (LQOI Issue #48)
+function linear15test(model::MOI.ModelLike, config::TestConfig)
+    atol = config.atol
+    rtol = config.rtol
+    # minimize 0
+    # s.t. 0 == 0
+    #      x == 1
+    @test MOI.supports(model, MOI.ObjectiveFunction{MOI.ScalarAffineFunction{Float64}}())
+    @test MOI.supports(model, MOI.ObjectiveSense())
+    @test MOI.supportsconstraint(model, MOI.VectorAffineFunction{Float64}, MOI.Zeros)
+
+    MOI.empty!(model)
+    @test MOI.isempty(model)
+
+    x = MOI.addvariables!(model, 1)
+    # Create a VectorAffineFunction with two rows, but only
+    # one term, belonging to the second row. The first row,
+    # which is empty, is essentially a constraint that 0 == 0.
+    c = MOI.addconstraint!(model,
+        MOI.VectorAffineFunction(
+            MOI.VectorAffineTerm.(2, MOI.ScalarAffineTerm.([1.0], x)),
+            zeros(2)
+        ),
+        MOI.Zeros(2)
+    )
+
+    MOI.set!(model, MOI.ObjectiveFunction{MOI.ScalarAffineFunction{Float64}}(), MOI.ScalarAffineFunction(MOI.ScalarAffineTerm.([0.0], x), 0.0))
```

Comment:
Like should be wrapped here to follow JuMP style guide :)

Reply:
done!